### ISSUE TITLE
merge the odekro code into master

### ISSUE THREAD

**Issue by mhl** [mysociety/pombola#1750]
Once https://github.com/mysociety/pombola/pull/1742 is merged into `merge-current-rebased`, I believe it should be fine to merge the `merge-current-rebased` branch into `master`. This should only be done after the following, however:

- [x] A final check of: `git diff master...merge-current-rebased` to check that nothing outside the `pombola.ghana` application will have its behaviour changed.
- [x] Creating the merge commit
- [x] Make sure the tests pass on the merge commit
- [x] Deploying the merge commit to staging for each of the four production Pombola sites, and testing that site. (That should include running the hansard parser for Kenya for a couple of sittings to make sure that still works OK.)

There may be other sensible checks to add to that list - please add them if you think of any.

I'll email Ian an update on our progress.

**Comment by duncanparkes** [comment]
I've rebased merge-current-rebased onto master and pushed it as post-ghana-merge. The tests are passing, and I've checked over the changes and I don't think they'll change any behaviour outside of Ghana.

**Comment by duncanparkes** [comment]
The following vhosts now have this code deployed:

mzalendo.staging.mysociety.org
zw.mzalendo.dev.mysociety.org
nigeria.pombola.staging.mysociety.org
za-pombola-dassie.staging.mysociety.org
